### DISCUSSION TITLE
What's the best way to delete expired cached files?

### DISCUSSION THREAD

**Discussion by josneville** [PrefectHQ/prefect#10340] [Q&A]
I'm using prefect for a pandas dataframe ETL pipeline. At each stage, we cache the results and expire them after a day. This system works great, re-runs happen fast. But our cache folder is getting a bit large. I would like to delete the cached files that have expired, is there any command I can use to do that?

**Comment by discdiver** [comment]
Prefect doesn't have a special command to delete cached files, but you can create and run a script that delete them. That's what I've heard other users have done.
